query_engine/Cargo.toml

```diff
@@ -25,10 +25,12 @@ workspace = true
 workspace = true
 
 [dependencies]
+arc-swap = "1.4.0"
```

Comment:
Maybe we should add it as a workspace dependancy because it has been used by other modules too.